crates/sui-protocol-config/src/lib.rs

```diff
@@ -2490,9 +2501,17 @@ impl ProtocolConfig {
                     }
                     cfg.feature_flags.enable_coin_deny_list_v2 = true;
                 }
-                54 => {
+                53 => {
                     // Do not allow bridge committee to finalize on mainnet.
                     cfg.bridge_should_try_to_finalize_committee = Some(chain != Chain::Mainnet);
+
+                    // Turns on shared object congestion control on testnet.
+                    if chain != Chain::Mainnet {
+                        cfg.max_accumulated_txn_cost_per_object_in_checkpoint = Some(100);
```

Comment:
should we rename this, since per-commit and per-checkpoint are no longer the same thing?